<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>734</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>734</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>My JotNote</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>KENNETH-build-desktop-Qt_4_8_1_for_Desktop_-_MinGW__Qt_SDK__Debug/tasks.jpg</normaloff>KENNETH-build-desktop-Qt_4_8_1_for_Desktop_-_MinGW__Qt_SDK__Debug/tasks.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>559</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textDue">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textSchedule">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textDynamic">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>601</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSomething">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>530</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Slot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushModify">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Modify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushRemove">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSearch">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>530</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushAdd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>530</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushHistory">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>530</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>History</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuToDo_List">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionNeon"/>
    <addaction name="actionSakura"/>
    <addaction name="actionKona"/>
    <addaction name="actionCheckered"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHelp_Info"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionClear_Cache"/>
   </widget>
   <widget class="QMenu" name="menuMy_JotNote">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>@@@@@@@@</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>@@@@@@@@</string>
    </property>
   </widget>
   <addaction name="menuMy_JotNote"/>
   <addaction name="menuFile"/>
   <addaction name="menuToDo_List"/>
   <addaction name="menuSettings"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNeon">
   <property name="text">
    <string>Neon</string>
   </property>
  </action>
  <action name="actionSakura">
   <property name="text">
    <string>Sakura</string>
   </property>
  </action>
  <action name="actionKona">
   <property name="text">
    <string>Kona</string>
   </property>
   <property name="toolTip">
    <string>Kona</string>
   </property>
  </action>
  <action name="actionCheckered">
   <property name="text">
    <string>Checkered</string>
   </property>
  </action>
  <action name="actionHelp_Info">
   <property name="text">
    <string>Help Info</string>
   </property>
  </action>
  <action name="actionClear_Cache">
   <property name="text">
    <string>Clear Cache</string>
   </property>
  </action>
  <action name="actionAs">
   <property name="text">
    <string>as</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
